src/screens/moviesList/styles.ts

```diff
@@ -0,0 +1,11 @@
+import colors from '@constants/colors';
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  heading: {
```

Comment:
add this in global styles as you are using this multiple times

Reply:
This style is subjected to be removed so please ignore it.